playground/infrastructure/cd_helper.py

```diff
@@ -22,16 +24,30 @@ class CDHelper:
     It is used to save beam examples/katas/tests and their output on the Google Cloud.
     """
 
-    def __init__(self):
-        self.ci_helper = CIHelper()
+    def get_examples_and_store(self):
+        """ Store beam examples and their output in the Google Cloud.
+        """
+        root_dir = os.getenv("BEAM_ROOT_DIR")
```

Comment:
I still think that this is a bad idea to retrieve environment variables so implicitly inside the class method. Why don't we just pass it as an argument for the function and retrieve such envs where it will be executed?

Reply:
Changed.